lhotse/cut/set.py

```diff
@@ -606,80 +605,268 @@ def save(cut: DataCut):
+            │ Features available:       │ 0        │
+            ├───────────────────────────┼──────────┤
+            │ Supervisions available:   │ 102222   │
+            ╘═══════════════════════════╧══════════╛
+            Speech duration statistics:
+            ╒══════════════════════════════╤══════════╤═══════════════════════════╕
+            │ Total speech duration        │ 64:59:51 │ 81.88% of recording       │
+            ├──────────────────────────────┼──────────┼───────────────────────────┤
+            │ Total speaking time duration │ 74:33:09 │ 93.91% of recording       │
+            ├──────────────────────────────┼──────────┼───────────────────────────┤
+            │ Total silence duration       │ 14:23:12 │ 18.12% of recording       │
+            ├──────────────────────────────┼──────────┼───────────────────────────┤
+            │ Single-speaker duration      │ 56:18:24 │ 70.93% (86.63% of speech) │
+            ├──────────────────────────────┼──────────┼───────────────────────────┤
+            │ Overlapped speech duration   │ 08:41:28 │ 10.95% (13.37% of speech) │
+            ╘══════════════════════════════╧══════════╧═══════════════════════════╛
+            Speech duration statistics by number of speakers:
+            ╒══════════════════════╤═══════════════════════╤════════════════════════════╤═══════════════╤══════════════════════╕
+            │ Number of speakers   │ Duration (hh:mm:ss)   │ Speaking time (hh:mm:ss)   │ % of speech   │ % of speaking time   │
+            ╞══════════════════════╪═══════════════════════╪════════════════════════════╪═══════════════╪══════════════════════╡
+            │ 1                    │ 56:18:24              │ 56:18:24                   │ 86.63%        │ 75.53%               │
+            ├──────────────────────┼───────────────────────┼────────────────────────────┼───────────────┼──────────────────────┤
+            │ 2                    │ 07:51:44              │ 15:43:28                   │ 12.10%        │ 21.09%               │
+            ├──────────────────────┼───────────────────────┼────────────────────────────┼───────────────┼──────────────────────┤
+            │ 3                    │ 00:47:36              │ 02:22:47                   │ 1.22%         │ 3.19%                │
+            ├──────────────────────┼───────────────────────┼────────────────────────────┼───────────────┼──────────────────────┤
+            │ 4                    │ 00:02:08              │ 00:08:31                   │ 0.05%         │ 0.19%                │
+            ├──────────────────────┼───────────────────────┼────────────────────────────┼───────────────┼──────────────────────┤
+            │ Total                │ 64:59:51              │ 74:33:09                   │ 100.00%       │ 100.00%              │
+            ╘══════════════════════╧═══════════════════════╧════════════════════════════╧═══════════════╧══════════════════════╛
+        """
+        from tabulate import tabulate
```

Comment:
Shall we remind users about the installation in case it is not installed?

Reply:
Yes, good point. I will also add it in the setup.py.